<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Profile</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_1"/>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>Email Address:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="lineEdit_9"/>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="lineEdit_7"/>
        </item>
        <item row="8" column="3">
         <widget class="QLineEdit" name="lineEdit_8"/>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
        <item row="10" column="3">
         <widget class="QLineEdit" name="lineEdit_10"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>Phone Number:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_1">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>First Name:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
        <item row="6" column="3">
         <widget class="QLineEdit" name="lineEdit_6"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>Weight:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>Last Name:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>SSN:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>Age:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>Medication List:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Appointment</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="18" column="0">
         <widget class="QLineEdit" name="lineEdit">
          <property name="maximumSize">
           <size>
            <width>589</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QLineEdit" name="lineEdit_11"/>
        </item>
        <item row="18" column="3">
         <widget class="QLineEdit" name="lineEdit_15">
          <property name="maxLength">
           <number>300</number>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Cancel Appointment</string>
          </property>
         </widget>
        </item>
        <item row="19" column="3">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Cancel Appointment</string>
          </property>
         </widget>
        </item>
        <item row="19" column="4">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Cancel Appointment</string>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Cancel Appointment</string>
          </property>
         </widget>
        </item>
        <item row="18" column="4">
         <widget class="QLineEdit" name="lineEdit_16"/>
        </item>
        <item row="17" column="0" colspan="4">
         <widget class="QLabel" name="label_11">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 18pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>Appointments: { Date | Start Time | End Time}</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="label_18">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt Lucida Calligraphy;</string>
          </property>
          <property name="text">
           <string>Start Time:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLabel" name="label_19">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt Lucida Calligraphy;</string>
          </property>
          <property name="text">
           <string>End Time:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_17">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt Lucida Calligraphy;</string>
          </property>
          <property name="text">
           <string>Date:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="9" colspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QCalendarWidget" name="calendarWidget_2"/>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt Lucida Calligraphy</string>
             </property>
             <property name="text">
              <string>Next Appointment Highlighted in Gray: </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3" colspan="2">
         <widget class="QLabel" name="label_15">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt Lucida Calligraphy;</string>
          </property>
          <property name="text">
           <string>Schedule Appointment: Max 4</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
        <item row="4" column="3">
         <widget class="QTimeEdit" name="timeEdit"/>
        </item>
        <item row="6" column="3">
         <widget class="QTimeEdit" name="timeEdit_2"/>
        </item>
        <item row="7" column="3" rowspan="2">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Schedule:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Billing Info</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>Insurance Number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_12">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>Credit Card Number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_13">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt &quot;Lucida Calligraphy&quot;;</string>
          </property>
          <property name="text">
           <string>Billing Amount:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_14">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 125, 132);
font-weight: bold;
font: 12pt Lucida Calligraphy;</string>
          </property>
          <property name="text">
           <string>Payment Amount:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_17"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>Pay</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
